Update README.md (#10)
vagelisp/Thessaloniki-WordPress-Meetup-Block-Theme: README.md

# English version

> _Disclaimer: The contents of this section is auto translated for now._

Reviewer: Πιστεύω πως μπορεί πλέον να αφαιρεθεί το disclaimer 💪

What is a block theme, and how is it made? What is Full Site Editing (FSE) and block patterns? How can I make my own custom blocks? As part of the Thessaloniki WordPress Meetup, a series of practical workshops are organized, where we will answer these questions, write code, collaborate as a team using Git, Slack, and other tools, with the ultimate goal of creating a complete block theme!

The block theme will live, and be developed in this repository, which will be open for anyone to contribute.

## Tools
To contribute to the development of the theme but also to create your own block themes, custom blocks, patterns, etc. you will need some tools.

### General Tools
#### WordPress account
To be able to publish themes and plugins to the WordPress directories, you will need to have a wordpress.org account. Also with a wordpress.org account, you will be able to contribute to WordPress, get help in the support forums, rate and comment on themes and plugins.

To create an account on wordpress.org click [here](https://login.wordpress.org/register?locale=en).

#### Slack

> slack is a communication platform. It provides instant discussion via text messages or online calls, and many additional features such as sharing images, code, etc. By signing up to the Hellenic WordPress Community Slack, you will have access to all discussions about WordPress, local WordPress Meetups and WordCamps, and other topics.

Reviewer: Δεδομένου ότι το Slack είναι αναφορά σε εμπορική επωνυμία καλύτερα να ενσωματωθεί με κεφαλαίο. Ξέρω πως δεν αφορά άμεσα δική σου αλλαγή, αλλά στα πλαίσια του review έπεσε εκεί το μάτι μου 🙈 
```suggestion
Slack is a communication platform. It provides instant discussion via text messages or online calls, and many additional features such as sharing images, code, etc. By signing up to the Hellenic WordPress Community Slack, you will have access to all discussions about WordPress, local WordPress Meetups and WordCamps, and other topics.

```

To sign up for Slack follow this [guide](https://wpgreece.org/wordpress-greece-slack/).

### Code development & management tools
#### VS Code
Visual Studio Code, is an [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment), which integrates various useful tools to facilitate code development, and debugging (e.g. syntax highlighting, code autocompletion , debug tools, etc.) and is provided for free.

To download VS Code click [here](https://code.visualstudio.com/Download)

#### Git
Git is a version control system. It allows us to know who, when, and what changes made to a file, while facilitating the collaboration of many people working on the same files.

To download Git click [here](https://git-scm.com/).
You can see a short but comprehensive guide with an explanation of basic Git concepts, and usage examples [here](https://rogerdudler.github.io/git-guide/).
[Here](https://docs.github.com/en/get-started/quickstart/hello-world) you will find a basic example of how to use GitHub.

#### Node JS & npm
Node.js is a software development platform built with Javascript.

> npm is a package manager for the JavaScript programming language. A package manager is a collection of tools that automate the process of installing, upgrading, and removing computer programs.

Reviewer: Nice catch! 👍

To download Node JS click [here](https://nodejs.org/en/download/) (npm is included and will be installed with Node JS).

### Local development environment
A local development environment is basically the installation of a web server on your computer, which you can use to easily create WordPress instances, and proceed with the development, execution, and testing of code.

Although there are many alternatives for creating a local development environment, we recommend using [Laragon](https://laragon.org/) or [Local](https://localwp.com/)

## Workshops Schedule
The schedule for the workshops is as follows:

**[27th Thessaloniki WordPress Meetup - 05 November 2022](https://www.meetup.com/wordpress_skg/events/289306513/)**
- Brief presentation of the project.
- What is a theme, and what are the differences between a classic and a block theme.
- Presentation, and assistance with installation of necessary tools.
- Creation of the basic framework of the block theme.
 

## Frequently Asked Questions (FAQ)
### What will be the purpose of this theme?

> It is going to be a general purpose theme, without targeting any specific type of users, or functions. It will be created with the goal to explore and practice with the tools, technologies and practices necessary to create a custom block-based theme. Also, once this theme will be complete, it will be published in the WordPress theme directory.

Reviewer: Μήπως να κάνουμε ένα μικρό rephrase εδώ, για να διώξουμε τις πολλαπλές αναφορές στο practice;
```suggestion
It is going to be a general purpose theme, without targeting any specific type of users, or functions. It will be created with the goal to explore and actually use the tools, the technologies and thepractices that are necessary to create a custom block-based theme. Also, once this theme will be complete, it will be published in the WordPress theme directory.

```

Reviewer 2: Μήπως κάπως έτσι?

It will be a general-purpose theme without targeting specific types of users or functions. We aim to explore and use the tools, technologies, and practices necessary to develop a custom block-based theme. Also, once this theme is complete, it will be published in the WordPress theme directory.

> It is going to be a general purpose theme, without targeting any specific type of users, or functions. It will be created with the goal to explore and practice with the tools, technologies and practices necessary to create a custom block-based theme. Also, once this theme will be complete, it will be published in the WordPress theme directory.

Reviewer 2: It will be a general-purpose theme without targeting specific types of users or functions. We aim to explore and use the tools, technologies, and practices necessary to develop a custom block-based theme. Also, once this theme is complete, it will be published in the WordPress theme directory.
